Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -25,6 +25,7 @@ class BrowserViewController: UIViewController {
     private var searchController: SearchViewController?
     private var webViewContainer: UIView!
     private let uriFixup = URIFixup()
+    private var screenshotHelper: ScreenshotHelper!
```

Comment:
I can make it optional, but I'm not sure I get your point. If it's optional and we do forget to set it, the tabs tray will look broken (no thumbnails at all), so we might as well just crash so we can get a stacktrace and fix it.